Libs/MRML/Core/vtkMRMLAbstractViewNode.h

```diff
@@ -110,9 +110,19 @@ class VTK_MRML_EXPORT vtkMRMLAbstractViewNode
   /// \sa GetVisibility()
   /// \sa IsViewVisibleInLayout()
   /// \sa vtkMRMLLayoutNode::SetViewArrangement()
+//TODO: Remove
   virtual int IsMappedInLayout();
   virtual void SetMappedInLayout(int value);
 
+  /// Get parent layout node.
+  /// Default is no reference, meaning that the view is mapped in the main layout.
```

Comment:
I think this should be "...meaning that the view is managed by the main layout".

Even if a layout manager manages a view, in a particular layout a view may or may not be mapped.